recipes/enpt_enmapboxapp/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "enpt_enmapboxapp" %}
+{% set version = "0.7.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/enpt_enmapboxapp-{{ version }}.tar.gz
+  sha256: ada1c75c6c9a562bde374542aff4774b45e22e6f62f9f5757966cb2905e9cb89
+
+build:
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - importlib-metadata
+    - packaging
```

Comment:
These don't seem to be needed at package build time?
```suggestion
```

Reply:
My apologies, I now see these hiding at the bottom of `setup.py`, so I was wrong.

Reply:
Actually, they should only be needed if the `enpt` package is installed.

Reply:
Ok fixed, all green now.